### PR TITLE
Security test fix

### PR DESCRIPTION
**Pull request by Ostrokrzew**
Fix for creating partition on partition:
now partition would be created on parent device.

Without this change every execution of 'test_user_cli' from security tests would fail right after the start.

Signed-off-by: Slawomir Jankowski <slawomir.jankowski@intel.com>

### CODE REVIEW

File: test/functional/tests/security/test_security_user.py [hunk 1]
```diff
@@ -41,8 +41,8 @@ def test_user_cli():
         cache_dev = cache_dev.partitions[0]
         core_dev = TestRun.disks['core']
         core_dev.create_partitions([Size(1, Unit.GibiByte), Size(256, Unit.MebiByte)])
-        core_dev = core_dev.partitions[0]
         core_dev1 = core_dev.partitions[1]
+        core_dev = core_dev.partitions[0]
```
Comment (katlapinka) [inline test/functional/tests/security/test_security_user.py]: What do you think about naming these variables like core_disk/cache_disk and those with assigned partitions like core_dev/cache_dev?

Comment (robertbaldyga) [reply test/functional/tests/security/test_security_user.py]: Why don't just name them cache_dev/core_dev and cache_part/core_part?

Comment (Ostrokrzew) [reply test/functional/tests/security/test_security_user.py]: That's good idea. Done.

**Comment by Ostrokrzew** [comment]
retest this please

**Pull request merged**